includes/admin/give-metabox-functions.php

```diff
@@ -1694,3 +1694,84 @@ function give_repeater_field_set_editor_id( $field_name, $field ) {
 }
 
 add_filter( 'give_get_field_name', 'give_repeater_field_set_editor_id', 10, 2 );
+
+/**
+ * Output Donation form radio input box.
+ *
+ * @since  2.1.3
+ *
+ * @param  array $field {
+ *                              Optional. Array of radio field arguments.
+ *
+ * @type string $id Field ID. Default ''.
+ * @type string $style CSS style for input field. Default ''.
+ * @type string $wrapper_class CSS class to use for wrapper of input field. Default ''.
+ * @type string $value Value of input field. Default ''.
+ * @type string $name Name of input field. Default ''.
+ * @type string $description Description of input field. Default ''.
+ * @type array $attributes List of attributes of input field. Default array().
+ *                                               for example: 'attributes' => array( 'placeholder' => '*****', 'class'
+ *                                               => '****' )
+ * @type array $options List of options. Default array().
+ *                                               for example: 'options' => array( 'enable' => 'Enable', 'disable' =>
+ *                                               'Disable' )
+ * }
+ * @return void
+ */
+function give_donation_form_goal( $field ) {
+	global $thepostid, $post;
+
+	$thepostid              = empty( $thepostid ) ? $post->ID : $thepostid;
+	$field['style']         = isset( $field['style'] ) ? $field['style'] : '';
+	$field['wrapper_class'] = isset( $field['wrapper_class'] ) ? $field['wrapper_class'] : '';
+	$field['value']         = give_get_field_value( $field, $thepostid );
+	$field['name']          = isset( $field['name'] ) ? $field['name'] : $field['id'];
+
+
+	printf(
+		'<fieldset class="give-field-wrap %s_field %s">',
+		esc_attr( $field['id'] ),
+		esc_attr( $field['wrapper_class'] )
+	);
+
+	printf(
+		'<span class="give-field-label">%s</span>',
+		esc_html( $field['name'] )
+	);
+
+	printf(
+		'<legend class="screen-reader-text">%s</legend>',
+		esc_html( $field['name'] )
+	);
+	?>
+
+    <ul class="give-radios">
+		<?php
+		foreach ( $field['options'] as $key => $value ) {
+			printf(
+				'<li><label><input name="%s" value="%s" type="radio" style="%s" %s %s /> %s </label></li>',
+				give_get_field_name( $field ),
+				esc_attr( $key ),
+				esc_attr( $field['style'] ),
+				checked( esc_attr( $field['value'] ), esc_attr( $key ), false ),
+				give_get_custom_attributes( $field ),
+				esc_html( $value )
+			);
+		}
+		?>
+    </ul>
+
+	<?php
+	/**
+	 * Action to add HTML after donation form radio button is display and before description
```

Comment:
Missing period at end of sentence breaks WPCS rules. Please @raftaar1191 ensure you're using wpcs so you don't make simple mistakes.